qiling/os/windows/dlls/kernel32/fileapi.py

```diff
@@ -303,7 +305,7 @@ def hook_GetVolumeInformationW(ql, address, params):
         pt_volume_name = params["lpVolumeNameBuffer"]
         if pt_volume_name != 0:
             # TODO implement
-            volume_name = ("AAAABBBB"+"\x00").encode("utf-16le")
+            volume_name = ("AAAABBBB" + "\x00").encode("utf-16le")
```

Comment:
do we need this in config so user can change it ?

Reply:
7de3cd5